### DISCUSSION TITLE
Experiencia con Test Driven Development y uso de REST-APIs

### DISCUSSION THREAD

**Discussion by albertcastaned** [Solify-IT/docs#8] [General]
Por favor compartir como comentario su experiencia con Test Driven Development (TDD) o con pruebas de software en general. También compartir si tienen experiencia creando o utilizando REST-APIs.

Si no tienen experiencia con alguno de estos por favor indicar si estarían dispuestos a aprender sobre ellos para el proyecto.

**Comment by albertcastaned** [comment]
TDD
He aplicado test driven development con unit tests e integration tests.


REST-APIs
Creado y utilizado REST-APIs. Principalmente con Django-REST

**Comment by Topi99** [comment]
**TDD**:  prácticamente igual que @albertcastaned 

**REST-APIs**: Creado y suado REST-APIs con NodeJS puro, Express, Django-REST, Falcon y Echo (Go).

**Comment by arcosmarce** [comment]
No tengo experiencia pero estoy dispuesta a aprender
